Syndemics/R/fetch_sas.R

```diff
@@ -1,13 +1,16 @@
 #' MA DPH PHDW Script Pull
 #' Fetches most recent .SAS files for MA DPH work and saves them to a SAS/ subdirectory
 #'
-#' @importFrom httr GET
+#' @import httr
 #'
 #' @export
 fetch_sas <- function(){
   if(!dir.exists("SAS")) dir.create("SAS")
+  req <- httr::GET("https://api.github.com/repos/SyndemicsLab/PHD/git/trees/main?recursive=1")
```

Comment:
This only pulls from main. Just noting in the condition that you might want to pull sas from a different branch like PreVenT?

Reply:
I can add this condition, but as this is a RESPOND package + CRC, does PreVenT even need to be included?